index.html

```diff
@@ -160,14 +173,16 @@ <h3>Skills</h3>
           Get in touch
         </button>
       </form>
-      <ul class="social2">
-          <li><a href="https://twitter.com/NoeSeguraL"><img src="img/twitter.png"></a></li>
-          <li><a href="https://www.linkedin.com/in/nseguralu/"><img src="img/Linkedin icon.png"></a></li>
-          <li><a href="https://medium.com/@nseguralu"><img src="img/medium.png"></a></li>
-          <li><a href="https://github.com/Nkaleth"><img src="img/github.png"></a></li>
-          <li><a href="https://angel.co/u/nilton-segura"><img src="img/angellist.png"></a></li>
-        </ul>
-        <p class="my_mail">nseguralu@gmail.com</p>
     </section> 
+    <footer class="footer">
+      <ul class="social2">
```

Comment:
You seem to have missed this horizontal rule that is before the social icons. Kindly include it. Also, notice how the background image does enter the area below the line.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206628306-124612f6-81ea-4d8f-a59b-12db349c4d87.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/206628318-f991054c-bf8d-466e-9c4f-16bf4401df5b.png)